package/MDAnalysis/analysis/base.py

```diff
@@ -220,7 +224,176 @@ def __setstate__(self, state):
         self.data = state
 
 
-class AnalysisBase(object):
+class AnalysisCollection(object):
```

Comment:
I don't think we need to inherit from object any more either (it's implicit)

Reply:
agree